apheleia.el

```diff
@@ -389,18 +389,29 @@ as its sole argument."
                       ;; changes, and 2 if error.
                       (memq status '(0 1)))))))
 
-(defun apheleia--run-formatter (command callback)
-  "Run a code formatter on the current buffer.
-The formatter is specified by COMMAND, a list of strings or
-symbols (see `apheleia-format-buffer'). Invoke CALLBACK with one
-argument, a buffer containing the output of the formatter.
-
-If COMMAND uses the symbol `file' and the current buffer is
-modified from what is written to disk, then don't do anything."
+(defun apheleia--format-command (command &optional stdin)
+  "Format COMMAND into a shell-ccmd and list of file paths.
```

Comment:
Is `ccmd` a typo or just an abbreviation I'm not familiar with?

Reply:
Typo. It was originally just shell-command but the documentation linter complained it wasn't uppercase and I had to condense it (clearly didn't do a good job) I'll fix that now.